django/core/handlers/base.py

```diff
@@ -113,6 +113,7 @@ def get_response(self, request):
         urlconf = settings.ROOT_URLCONF
         urlresolvers.set_urlconf(urlconf)
         resolver = urlresolvers.RegexURLResolver(r'^/', urlconf)
+        rendered = False
```

Comment:
After iterating over the items in `_template_response_middleware` Django explicitly renders the response (which is correct). To prevent double rendering, could you update `rendered` to `True` after the rendering in line 177, please :)